assets/js/blocks/breadcrumbs/edit.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export interface Attributes {
+	className?: string;
+}
+
+const Edit = () => {
+	const blockProps = useBlockProps( {
+		className: 'woocommerce wc-block-breadcrumbs',
+	} );
+
+	return (
+		<div { ...blockProps }>
+			<a href="/">
```

Comment:
I would recommend to disable the pointer events on this element. I like the idea of showcasing the link in Editor, but when you click it, it redirects to the frontend for a second and goes back to Editor, which may feel strange to the users. I can imagine scenario the user clicks that by accident when customizing the page.

You can create `editor.scss` file and import it in `edit.tsx`, so the styles won't leak to the frontend: [example](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/atomic/blocks/product-elements/rating/editor.scss).

Reply:
Good call, [updated](https://github.com/woocommerce/woocommerce-blocks/pull/8222/commits/70b1b3d0cfcd6543750b9523ce10b495dc352b93)!

Reply:
We can use the `Disabled` component here to prevent the interaction with the component https://github.com/woocommerce/woocommerce-blocks/blob/ded8728ac2fcaf9731eb606fde94d1350d67a816/assets/js/blocks/catalog-sorting/edit.tsx#L26-L28

Reply:
Thanks! Updated to rely on the `<Disabled>` component so we are more aligned with the other blocks.